db/migrate/20240731191344_change_primary_identifier_for_security.rb

```diff
@@ -0,0 +1,7 @@
+class ChangePrimaryIdentifierForSecurity < ActiveRecord::Migration[7.2]
+  def change
+    rename_column :securities, :symbol, :ticker
+    remove_column :securities, :isin, :string
+    rename_column :security_prices, :isin, :ticker
+  end
+end
```

Comment:
This is largely a simplification of how we identify securities internally.  My original thinking was that `isin` codes would provide globally unique identification for securities and help us avoid duplicates, but after doing some research on various data providers, this is not viable.

For example, Plaid provides an `isin` code for each security, while Polygon (a popular stock price provider) doesn't provide an isin code at all.  Instead, it provides a FIGI code, which is the new globally unique identifier standard for all types of financial instruments that is not subject to change when a company reclassifies, splits, moves exchanges, etc. (isin can change in rare cases).

While a "ticker" symbol isn't globally unique, it fits our domain model well.  Each ticker represents a company + exchange pair, which results in a unique price.

Reply:
The main issue is that tickers are for specific companies. All ETF and active funds will have an ISIN instead. 
Tickers are useful but ISIN are the standard for looking up values of an investment and are widely available on the Key Investor Information or popular web sites such as morningstar. 
Without the ISIN, how will you add investments which are not companies?